Zend/zend_object_handlers.c

```diff
@@ -62,7 +62,7 @@
   called, we cal __call handler.
 */
 
-ZEND_API void rebuild_object_properties(zend_object *zobj) /* {{{ */
+ZEND_API void rebuild_object_properties_internal(zend_object *zobj) /* {{{ */
```

Comment:
As the API is changed anyway, it makes sense to return ``HashTable`` and avoid extra load of `` zobj->properties`` in ``zend_std_get_properties[_ex]``.